### ISSUE TITLE
Upgrade instructions

### ISSUE THREAD

**Issue by jmatsushita** [frappe/frappe_docker#13]
Hi there,

Thanks a lot for making an official docker image with compose instructions! I've been using my own version so far based on previous discussions on the forum and I'd like to migrate to this one.

One of the things that have been tricky so far has been to update to new versions of erpnext. Is there a recommended way to upgrade, ideally with the docker way i.e. by upgrading the relevant containers and letting the entrypoint run the upgrades. Could you post instructions on the README about this?

Very best,

Jun

**Comment by vishalseshagiri** [comment]
Hey @jmatsushita. Well all you need to do is either attach to the container or exec -it into it and do a ```bench update```

**Comment by vishalseshagiri** [comment]
Or using the dbench commands you can do a ```./dbench update``` from outside the container this  should update all your apps. I am assuming this is what you were looking for.

**Comment by pklapperich** [comment]
While looking for the upgrade instructions, I found this issue before I found the actual instructions and the docker image has been refactored such that `bench update` is no longer correct. Don't try that.

For anyone else arriving here from a web search, see the [Setup Options - Updating Images](https://github.com/frappe/frappe_docker/blob/main/docs/setup-options.md#updating-images) document and adjust the commands as necessary to fit your deployment. After upgrading the containers, make sure you run database migrations as described in [migrate site](https://github.com/frappe/frappe_docker/blob/main/docs/site-operations.md#migrate-site). You probably want to backup your database and sites-volume before running the migrations, just in case.
